jac-cloud/jac_cloud/plugin/implementation/api.py

```diff
@@ -56,6 +56,8 @@ class DefaultSpecs:
     auth: bool = True
     private: bool = False
     webhook: dict | None = None
+    tags: list[str] | None = None
+    allow_entry: bool | None = None
```

Comment:
Two quick ones:
- I'd name this `allow_node_entry` 
- can we default this to True instead of None?

Reply:
rename this to entry_type and the value should be an enum for ROOT, NODE, BOTH